backend/apps/sandbox/views.py

```
@@ -448,3 +448,23 @@ def perform_create(self, serializer):
+
+from .models import MaintainerScenario, MaintainerEvaluation
+from .serializers import MaintainerScenarioSerializer, MaintainerEvaluationSerializer
+
+class MaintainerScenarioViewSet(viewsets.ReadOnlyModelViewSet):
+    serializer_class = MaintainerScenarioSerializer
+    permission_classes = [permissions.IsAuthenticated]
+    queryset = MaintainerScenario.objects.all()
+
+class MaintainerEvaluationViewSet(viewsets.ModelViewSet):
+    serializer_class = MaintainerEvaluationSerializer
+    permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)